src/server/routes/api/document/index.js

```diff
@@ -518,7 +580,13 @@ http.get('/zip', function( req, res, next ){
  *             description: Download a zip file containing each Document represented as BioPAX in owl file format.
```

Comment:
Does the swagger need this (also in `/biopax/:id`)

```
/**
 * @swagger
 *
 * /api/document/zip/biopax:
 *   get:
 *     description: Download a single zip file containing every Document represented in Biological Pathway Exchange (BioPAX).
 *     parameters:
 *       - name: idMapping
 *         in: query
 *         description: ...
 *         required: false
 *         type: boolean
 *         allowEmptyValue: true
```

Reply:
@jvwong I just made the swagger updates. While testing them out I recognized that https://master.grounding.baderlab.org/ is not available.

Reply:
I had to reboot but its fine now